script/fe_prestop.sh

```diff
@@ -19,7 +19,8 @@ SELF_HOST=
 #left_node_lists node lists of except self.
 LEFT_NODE_LSITS=
 PROBE_LEADER_PODX_TIMEOUT=120 # at most 60 attempts
-STARROCK_HOME=${STARROCK_HOME:-"/opt/starrocks"}
+STARROCKS_ROOT=${STARROCKS_HOME:-"/opt/starrocks"}
+STARROCKS_HOME=$STARROCKS_ROOT/fe
```

Comment:
`STARROCKS_FE_HOME`

Reply:
this is starrocks script set, we follow the starrocks.